default.nix

```diff
@@ -57,6 +57,8 @@ in {
           # Ensures that clojure creates all the caches in our empty separate home directory
           export JAVA_TOOL_OPTIONS="-Duser.home=$tmp/home"
 
+          mkdir -p "$tmp/home/.gitlibs/libs"
```

Comment:
Why is this necessary?

Reply:
`locker.py` expects it to be there, and if there are 0 git dependencies then the clojure tooling never creates it.

Thinking about it though, maybe we want `locker.py` to handle this gracefully instead and maybe not even emit a `git` key in the lockfile then. WDYT?

Reply:
Yeah I think handling this in the python script would be better

Reply:
I'll do that instead and push it :)